client/my-sites/marketplace/pages/marketplace-plugin-install/index.tsx

```diff
@@ -264,10 +264,10 @@ const MarketplacePluginInstall = ( {
 					illustration="/calypso/images/illustrations/error.svg"
 					title={ null }
 					line={ translate(
-						"Your current plan doesn't allow plugin installation. Please upgrade to Business plan first."
+						"Your current plan doesn't allow plugin installation. Please upgrade to Pro plan first."
```

Comment:
This has created a path for a legacy plan to transit to the Pro plan, but we will leave it for now: p2-pdgrnI-xR#comment-1052